backend/app/api/run_management.py

```diff
@@ -31,6 +31,12 @@ def list_runs(
     runs = query.order_by(RunModel.start_time.desc()).limit(last_k).all()
     return runs
 
+@router.get("/{run_id}/", response_model=RunResponseSchema)
```

Comment:
There's an inconsistency here that would be good to fix now: RunResponseSchema doesn't have tasks, but RunStatusResponse does.
If you are using API with RunResponseSchema then let's add tasks to it. Tasks will contain the subworkflow details when applicable.